app/assets/javascripts/discourse/templates/preferences/users.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="control-group muting">
+  <label class="control-label">{{i18n 'user.users'}}</label>
+  <div class="controls category-controls">
+    <label>{{d-icon "d-muted" class="``icon"}} {{i18n 'user.muted_users'}}</label>
+    {{user-selector excludeCurrentUser=true usernames=model.muted_usernames class="user-selector"}}
+  </div>
+  <div class="instructions">{{i18n 'user.muted_users_instructions'}}</div>
+
+  {{#if siteSettings.ignore_user_enabled}}
+    <div class="controls category-controls">
+      <label>{{d-icon "eye-slash" class="icon"}} {{i18n 'user.ignored_users'}}</label>
+      {{user-selector excludeCurrentUser=true
+                      usernames=model.ignored_usernames class="user-selector"}}
+    </div>
+    <div class="instructions">{{i18n 'user.ignored_users_instructions'}}</div>
+  {{/if}}
+</div>
+
+{{plugin-outlet name="user-preferences-notifications" args=(hash model=model save=(action "save"))}}
+
+<br/>
```

Comment:
Not a big fan of <br /> to position elements. Please use css if possible. Less markup, easier to maintain, easier to customise.

Reply:
Totally 👍 I'd rather not touch the `plugin-outlet` for now.